addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/ExtensionWappalyzer.java

```diff
@@ -248,6 +242,11 @@ public void unload() {
         super.unload();
 
         getPscanExtension().getPassiveScannersManager().remove(passiveScanner);
+        View.getSingleton()
```

Comment:
This should only be done if there's view, also preferable to let the panel unload itself rather than exposing internals.

Reply:
That's all that's needed right? 
I mean there's only a panel if there's a View?